modules/restful_example/src/Plugin/resource/Articles__1_0.php

```diff
@@ -39,17 +41,55 @@ class Articles__1_0 extends ResourceEntity implements ResourceInterface {
   protected function publicFields() {
     return array(
       'id' => array(
-        'wrapper_method' => 'getIdentifier',
-        'wrapper_method_on_entity' => TRUE,
+        'property' => 'nid',
```

Comment:
minor,but I prefer the existing format of  ``$public_fields['id'] = array(..`` -- just because it make it easier to copy/ paster from one resource to another.